app/concepts/user/create.rb

```diff
@@ -0,0 +1,12 @@
+class User::Create < Trailblazer::Operation
+  include Model
+  model User
+
+  def process(params)
+    @model = User.new
```

Comment:
Is `model` undefined here? The model call on line 3 creates assigns `model` under the hood for you.